### ISSUE TITLE
--url command line parameter

### ISSUE THREAD

**Issue by KrisKA-360** [nwjs/nw.js#4963]
-url parameter seems to not run from command line !
nwjs-sdk-v0.15.2-win-x64

**Comment by KrisKA-360** [comment]
When package.json contains next parameter {"main" : http://nwjs.io}, external command line including --url parameter can not override "main" parameter.
It was possible in NW.js 0.12 and was very confortable.

**Comment by Christywl** [comment]
Yes, if the command line includes `package.json`, nwjs will always start App with `package.json` whatever if there is `--url` parameter in the command line.

@rogerwang , could you please confirm this?
